content/spin/v2/install.md

```diff
@@ -286,11 +286,9 @@ Then install plugins by name.
 
 Python:
 
-<!-- @selectiveCpy -->
+> Historic: You may have seen a `py2wasm` plugin in your travels. Please note, that `py2wasm` has since been replaced by `componentize-py`. (The `componentize-py` workflow is different from the historical `py2wasm` plugin-based approach.)
 
-```bash
-$ spin plugins install py2wasm --yes
-```
+The process of getting your system ready to write Wasm-powered Python applications using `componentize-py` can now be found in (the Building Spin Components in Python)[./python-components.md#componentize-py] area of the documentation.
```

Comment:
If you accept the suggestion above, remove this para - the suggestion is an attempt to emphasise that there is no longer _any_ "getting your system ready."

(If you do keep this para, could you double check the Markdown link syntax?  Elsewhere we seem to have the bracket types the other way round.)

Reply:
Thanks, removed this para and accepted the suggestion from above.